### ISSUE TITLE
Remove rendering_threads option in the CEF port.

### ISSUE THREAD

**Issue by emilio** [servo/servo#17882]
https://github.com/servo/servo/pull/17872 is removing its only user. Once it lands we can remove it without problems.

The field is in `ports/cef/types.rs`, I don't think this would require more changes than removing the field and related documentation.

**Comment by jdm** [comment]
Use `./mach build-cef` to build that code and avoid any compile errors.

**Comment by abahmed** [comment]
@highfive : assign me

**Comment by DarkDrek** [comment]
This was already done in #17905 so this issue can be closed. @jdm @emilio
